frontend/src/Auth/Auth.js

```diff
@@ -0,0 +1,41 @@
+import { Auth0Lock } from 'auth0-lock';
+
+// 'options' object contains Auth0 Lock-specific features to customize the Auth0 widget.
+const options = {
+	languageDictionary: {
+		title: 'Log In'
+	},
+	auth: {
+		sso: false
+	}
+};
+
+/**
+ * Auth0 {class}:
+ * creates an instance of Auth0Lock, using the Auth0 ClientID and Auth0 domain.
+ * login() / signUp() - call the Auth0 Lock 'show()' method with their respective default screens, while displaying the listed social connections options.
+ */
+
+export default class Auth0 {
+	lock = new Auth0Lock(
+		'uazofWna5PRTCwvCJA6vcbvVtRm8439M',
```

Comment:
Consider moving this (hopefully) client key into Redux in case it needs to be changed later. That way, you (or someone not familiar with it) won't have to spend a bunch of time hunting for it.